internal/errorhandling/docs.go

```diff
@@ -5,5 +5,11 @@ import "github.com/warpstreamlabs/bento/internal/docs"
 func Spec() docs.FieldSpecs {
 	return docs.FieldSpecs{
 		docs.FieldString(fieldStrategy, "The error handling strategy.").HasOptions("none", "reject").HasDefault("none"),
+		docs.FieldObject(fieldLog, "Configuration for global logging mmessage-level errors.").WithChildren(
+			docs.FieldBool(fieldLogEnabled, "Whether to enable message-level error logging.").HasDefault(false),
+			docs.FieldBool(fieldLogAddPayload, "Whether to add a failed message payload to an error log.").HasDefault(false),
+			docs.FieldFloat(fieldLogSamplingRatio, "Sets the ratio of errored messages within a batch to sample.").HasDefault(0).
```

Comment:
```suggestion
			docs.FieldFloat(fieldLogSamplingRatio, "Sets the ratio of errored messages within a batch to sample.").HasDefault(1).
```

Feel a default value of 1 makes more sense?

Reply:
Ahh that is fair. Initially I made a value of `0` only log the first error encountered but happy to make this `1`.

FYI payloads with a very large batch size and error rate will have their logs (potentially) flooded 🫠